spec/helpers/activity_helper_spec.rb

```diff
@@ -96,5 +96,22 @@
         expect(helper.show_activity_field?(activity: activity, step: "tied_status")).to be(false)
       end
     end
+
+    context "when the activity has a null .wizard_status field" do
+      it "shows all steps" do
+        activity = build(:activity_with_nil_wizard_status)
+
+        expect(helper.show_activity_field?(activity: activity, step: "identifier")).to be(true)
```

Comment:
Is it worth doing something like this:

```ruby
all_steps = Staff::ActivityFormsController::FORM_STEPS

all_steps.each do |step|
  expect(helper.show_activity_field?(activity: activity, step: step)).to be(true)
end
```

So the test will still ensure all steps are shown if we add additional ones?